CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Change log for Microsoft365DSC
 
 # UNRELEASED
-
+* AADConditionalAccessPolicy
+  * Add condition for empty External Guest/User include/exclude
```

Comment:
There is already an entry for the AADConditionalAccessPolicy header on line 25. Please add this item to that header.